### PR TITLE
Added new eventstream (Support AWS SQS).

### PR DESCRIPTION
**Pull request by felipeagger**
- New support for AWS SQS EventStream

### CODE REVIEW

File: sqs/sqs.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	if data, ok := opts[MessageGroupIdOption]; ok {
+		opt.msgGroupId = aws.String(data.(string))
+	}
+
+	if data, ok := opts[MessageDedupIdOption]; ok {
+		opt.msgDedupId = aws.String(data.(string))
+	}
+
+	return &opt
+}
```
Comment (italolelis) [inline sqs/sqs.go:90]: This is interesting. What about using [Funcional Options](https://www.sohamkamani.com/golang/options-pattern/) instead? Wdyt?

File: sqs/sqs_test.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/italolelis/outboxer"
+)
+
+func TestSQS_EventStream(t *testing.T) {
```
Comment (italolelis) [inline sqs/sqs_test.go:18]: We could potentially start using [table tests](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests). Wdyt?

**Comment by italolelis** [comment]
@felipeagger, thank you for your contribution. I left a few comments here to get your opinion, and maybe we can implement some of that.

File: sqs/sqs_test.go (hunk 2 above)
Comment (felipeagger) [reply sqs/sqs_test.go:18]: that's perfect! Adjusted!

File: sqs/sqs.go (hunk 1 above)
Comment (felipeagger) [reply sqs/sqs.go]: I didn't know, but I tried to implement it, Wdyt?

**Review by italolelis — APPROVED** [review]

**Comment by italolelis** [comment]
@felipeagger if you rebase the changes on master the linter errors should go away and we can merge this. Can you please rebase it? Thanks

**Comment by felipeagger** [comment]
> 

Done!

**Comment by italolelis** [comment]
@felipeagger thank you again for your contribution. I'm merging this!

**Pull request merged**